### DISCUSSION TITLE
URL Broken for OoTD CIv

### DISCUSSION THREAD

**Discussion by mathemagics** [aoe4world/explorer#26] [Bugs / Problems]
Url in the link is:
https://aoe4world.com/explorer/civs/orderofthedragons

Should be
https://aoe4world.com/explorer/civs/orderofthedragon

**Comment by robertvanhoesel** [comment]
Thanks. Fixed in https://github.com/aoe4world/web/commit/2dba1808b984f8c115df77d0b428997f780cd9fe

**Comment by mathemagics** [comment]
Nice! thanks!
